docs/application/dotnet/guides/nui/xaml/layout-define-example.md

```diff
@@ -5,8 +5,8 @@ You can use XAML to define the layout of user interface in your applications. To
 2. Load the XAML file
 3. Put the XAML file in the appropriate file directory
 
-> **Note**
-> 
+> [!NOTE]
+>
 > Tizen NUI XAML application platform provides a XAML application template and you can use XAML since **Tizen.NET API version 7**.
```

Comment:
I appreciate your update, @Seoyeon2Kim.

However, the one empty line with >, after the \[!NOTE] should be removed also.
Would you please check that point also?

```suggestion
> [!NOTE]
> Tizen NUI XAML application platform provides a XAML application template and you can use XAML since **Tizen.NET API version 7**.
```

Reply:
Oh!! okay, sure!
I'll remove all empty lines with >.